lib/omnibus/packagers/rpm.rb

```diff
@@ -494,7 +495,11 @@ def safe_version
       #   http://rpm.org/ticket/56
       #
       if version =~ /\-/
-        converted = version.gsub('-', '~')
+        if Ohai['platform_family'] == 'wrlinux'
```

Comment:
Right now we only have WRL5 but I can update this when I add support for WRL7 if necessary.